beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -418,7 +418,18 @@ func (vs *Server) blobSidecarsFromUnblindedBlock(block interfaces.SignedBeaconBl
 
 // broadcastReceiveChunkedBlock broadcasts a chunked block and handles its reception.
 func (vs *Server) broadcastReceiveChunkedBlock(ctx context.Context, req *ethpb.ChunkedBeaconBlock, root [32]byte) error {
-	return nil
+	block, err := blocks.NewSignedBeaconBlock(req.Block.Block)
+	if err != nil {
+		return errors.Wrap(err, "block construction failed")
+	}
+	if err := vs.P2P.BroadcastBlockChunks(ctx, req); err != nil {
+		return errors.Wrap(err, "broadcast failed")
+	}
```

Comment:
This creates a signed beacon block and then tries to send it as a chunked block.